src/components/DayPicker.jsx

```diff
@@ -149,6 +150,7 @@ export const defaultProps = {
   // month props
   renderMonthText: null,
   renderMonthElement: null,
+  renderWeekHeaderText: null,
```

Comment:
Not sure if this is better or worse, but did you consider making the default for this:

```
renderWeekHeaderText: day => <small>{day}</small>
```

?

That would collapse the number of conditionals from 2 (the ternary in the render prop and the default prop) to 1 (just the default prop).

But maybe it would be harder to see what the default is when scanning the component?

Reply:
ooh I like that idea! though I tend to see this pattern more where the function prop is optional and then some functionality is used in its place, rather than assigning that functionality as the default to the function prop—so I'm inclined to keep this as-is for consistency.